### ISSUE TITLE
Unable to install with pip3

### ISSUE THREAD

**Issue by ragreenburg** [jstockwin/py-pdf-parser#123]
**Bug Report**

Before submitting an issue, please ensure you have read our `CONTRIBUTING.md`, and follow the Code of Conduct.

Thanks for taking the time to report a bug. To help us fix it quickly, please include the following information:

Unable to install via pip as show in the documentation. Running `pip3 install py-pdf-parser` I get the error code 
```
    ERROR: Command errored out with exit status 1:
     command: /home/hank/Development/pdf_parse/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-j1z0oskj/wand/setup.py'"'"'; __file__='"'"'/tmp/pip-install-j1z0oskj/wand/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-j1z0oskj/wand/pip-egg-info
         cwd: /tmp/pip-install-j1z0oskj/wand/
    Complete output (18 lines):
    Traceback (most recent call last):
      File "/tmp/pip-install-j1z0oskj/wand/wand/api.py", line 180, in <module>
        libraries = load_library()
      File "/tmp/pip-install-j1z0oskj/wand/wand/api.py", line 135, in load_library
        raise IOError('cannot find library; tried paths: ' + repr(tried_paths))
    OSError: cannot find library; tried paths: []
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-j1z0oskj/wand/setup.py", line 8, in <module>
        from wand.version import VERSION
      File "/tmp/pip-install-j1z0oskj/wand/wand/version.py", line 45, in <module>
        from .api import libmagick, library
      File "/tmp/pip-install-j1z0oskj/wand/wand/api.py", line 198, in <module>
        distname, _, __ = platform.linux_distribution()
    AttributeError: module 'platform' has no attribute 'linux_distribution'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

```
I am trying to install on KDE Neon and using that and the dev version of py-pdf-parser give me the same error. Though it seems to be working on a Windows boot.

**Comment by jstockwin** [comment]
Hey @ragreenburg, thanks for the bug report. Apologies it didn't work first time!

This is actually this wand issue (https://github.com/emcconville/wand/issues/456), however it looks like they've already fixed it so we just need to upgrade our version.

In the meantime I will also enable dependabot on this repository, which should mean we always get up to date packages.

**Comment by jstockwin** [comment]
This was fixed in https://github.com/jstockwin/py-pdf-parser/pull/142, which I have just released as version 0.6.0 (https://pypi.org/project/py-pdf-parser/0.6.0/).

@ragreenburg I will close this issue, but please do let me know if it still doesn't work for some reason!

**Comment by ragreenburg** [comment]
Thanks mate, much appreciated!
